addons/csw/functions/fnc_staticWeaponInit.sqf

```diff
@@ -69,8 +69,8 @@ if (hasInterface && {!(_typeOf in GVAR(initializedStaticTypes))}) then {
     private _ammoActionPath = [];
     private _magazineLocation = getText (_configOf >> QUOTE(ADDON) >> "magazineLocation");
     private _condition = { //IGNORE_PRIVATE_WARNING ["_target", "_player"];
-        // If magazine handling is enabled or weapon assembly/disassembly is enabled we enable ammo handling
-        if ((GVAR(ammoHandling) == 0) && {!([false, true, true, GVAR(defaultAssemblyMode)] select (_target getVariable [QGVAR(assemblyMode), 3]))}) exitWith { false };
+        // If magazine handling and weapon assembly/disassembly are enabled we enable ammo handling
+        if ((GVAR(ammoHandling) == 0) || {!([false, true, true, GVAR(defaultAssemblyMode)] select (_target getVariable [QGVAR(assemblyMode), 3]))}) exitWith { false };
```

Comment:
I'm not sure about this,
before even if ammoHandling was off you could still always load a "CSW" static weapon

e.g. if you assemble a csw from 2 carried weapons then it would still be loadable
because otherwise you just have a empty weapon that you can't do anything with

Reply:
I'll figure something out. Though you might agree that ammo handling still being possible despite the setting is confusing.

Reply:
yeah I'd probably say it's mission-maker error
but docs also say
```
### 3.3 ammoHandling

- Whether or not you want to handle ammo using the CSW way. Does nothing if using defaultAssemblyMode
- Default: On
```

Reply:
IMO it shouldn't even be an option. This feature makes no sense without ammo handling.

Reply:
I think we should revert it to the previous behaviour, despite the setting being confusing.